setup/kubernetes/k3s.md

```diff
@@ -1,97 +1,123 @@
 ---
-title: "k3s"
-description: Set up a Coder deployment with a Ubuntu + k3s
+title: "K3s"
+description: Set up K3s on an Ubuntu machine to deploy Coder.
 ---
 
-k3s is a lightweight Kubernetes distribution that works well for single-node or
-multi-node clusters alike.
+This article will show you how to install K3s onto a new Ubuntu 20.04 LTS
+machine for use with Coder.
 
-> This guide only covers installation on a new Ubuntu 20.04 LTS machine. If you
-> are looking to install on a local machine or an existing host,
-> [Kind](./kind.md) or k3d may be a better choice.
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that works well
+for both single-node or multi-node clusters. This guide covers the installation
+of K3s onto a new Ubuntu 20.04 LTS machine, but if you want to install Coder on
+a local machine or an existing host, [Kind](./kind.md) or [k3d](https://k3d.io/)
+are better choices.
+
+> This installation method is not officially supported or tested by Coder. If
+> you have questions or run into issues, feel free to reach out using our
+> [community Slack channel](https://cdr.co/join-community).
 
 ## Prerequisites
 
-Before proceeding, please make sure that you have the following:
+Before proceeding, please make sure that:
+
+1. You have an **Ubuntu 20.04 machine**: This can be a bare metal or a virtual
+   machine. For cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr,
+   and DigitalOcean all work well.
 
-1. **Ubuntu 20.04 machine**: This can be a bare metal or virtual machine. For
-   cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr, DigitalOcean
-   work well. Make sure the machine specs satisfies Coder's
-   [resource requirements](../requirements.md)
-1. Network policy exposting ports 22, 80, 443, and 5522 (alternative SSH port)
-1. The following software installed on the machine:
+   Ensure that the machine's specs satisfies
+   Coder's [resource requirements](../requirements.md), since your experience
+   with Coder is dependent on your system specs.
+
+1. You have the following software installed on your machine:
    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
    - [helm](https://helm.sh/docs/intro/install/)
 
-### Resource allocation and performance
+1. Your network policy or firewall accepts incoming traffic on ports 22 (SSH),
+   80 (HTTP), 443 (HTTPS), 5349 (Networking v2 / TURN), and 8443 (Kubernetes
+   API).
```

Comment:
I don't think Kubernetes API needs to be exposed, you could mark it as optional if you want.